packages/bar/index.d.ts

```diff
@@ -99,6 +99,7 @@ declare module '@nivo/bar' {
         padding: number
 
         valueScale: Scale
+        indexedScale: Scale
```

Comment:
I think it would be better to call it `indexScale`, the scale type is a bandScale, and is used for indexes.